grunt/tasks/translate.js

```diff
@@ -12,7 +12,7 @@ module.exports = function(grunt) {
       "_exportLangFiles"
     ]);
 
-    grunt.registerTask("translate:import", "Import Language files and create a translated duplicte of a master Course.", [
+    grunt.registerTask('translate:import', 'Import Language files and create a translated duplicte of a master Course.', [
```

Comment:
could you change this to:
> 'Import translated language files'

Reply:
I think as this will follow on immediately from:
> translate:export    Export Course Data into Language files ready to be translated

It doesn't need a longer explanation.

I wonder if it would be useful to include a link to https://github.com/adaptlearning/adapt_framework/wiki/Course-Localisation here?

Reply:
@moloko I'm afraid it might look a bit messy, but definitely helpful.
Should I add a link to the description of "translate:export" like that:
> translate:export Export Course Data into Language files ready to be translated. More details:  https://github.com/adaptlearning/adapt_framework/wiki/Course-Localisation

?